### DISCUSSION TITLE
Image size

### DISCUSSION THREAD

**Discussion by bnlawrence** [chrisgrieser/shimmering-focus#205] [Q&A] [answered]
Hi. Thanks for this. I've just moved to it as I like the minimalism, but I think I need to handle the images differently.

I note that you have an image syntax something like `![[image.png|caption]]`, but all over the obsidian help pages, there is advice that you resize images with something like `![[image.png|widthxheight]]` - although maybe that's a theme specific thing?

Is there an alternative method for your theme? 

Your guidance says "images can be zoomed in by clicking and holding", but that isn't obviously working for me on a macbook (either for viewing or for saving) -  using an external mouse and keyboard (but maybe I'm being obtuse about what sort of click?)

Any additional suggestions on image behaviour very welcome (in practice, I usually want them actual size, or full width or whatever size of the column I have available).

**Comment by chrisgrieser** [comment, marked as answer]
> I note that you have an image syntax something like ![[image.png|caption]], but all over the obsidian help pages, there is advice that you resize images with something like ![[image.png|widthxheight]] - although maybe that's a theme specific thing?

yeah, the theme overrides the default Obsidian behavior. The advantage is, that you do not have to spend time on adding the proper image sizes, since the theme simply resizes all your images to the same width. You can adjust that width with the style settings plugin.

> Your guidance says "images can be zoomed in by clicking and holding", but that isn't obviously working for me on a macbook (either for viewing or for saving) - using an external mouse and keyboard (but maybe I'm being obtuse about what sort of click?)

If clicking and holding does not work to enlarge the image, please open a bug report (since the bug report form asks for the various kinds of information I need to fix it.)

If you'd like a different image behavior all-together, please open a feature request 🙂

**Comment by bnlawrence** [comment]
Many thanks Chris. I should think about things a bit more before I make any feature requests, and I'll play on another platform before concluding there is a bug :-).

I'll look into the styling.

**Reply by bnlawrence** [reply]
Just to say that somehow I seem to have both behaviours working now, using `![[blah|400]]` and the caption using `![caption](blah)`. Perfect.
